applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

```diff
@@ -353,7 +348,7 @@ def AssembleTestSuites():
     nightSuite.addTest(TRigidFaceTestWithImposeRigidMovementProcess('test_execution'))
     nightSuite.addTest(TRigidSphereFailing('test_execution'))
 
-    if (missing_external_dependencies == False):
+    if has_external_solvers_application:
         if (hasattr(KratosMultiphysics.ExternalSolversApplication, "FEASTSolver")):
```

Comment:
This could be further "improved" by using the skip decorator in the test factory. But i guess to see if the solver is available, the import has to happen.

Reply:
does this work as is without importing ExternalSolvers?

Reply:
Yes.

Reply:
But only because the import happens in some other tests that are imported above.